pkgs/development/python-modules/pgpy/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, cryptography
+, pyasn1
+, six
+, singledispatch
+}:
+
+buildPythonPackage rec {
+  pname = "PGPy";
+  version = "0.4.3";
+
+  src = fetchPypi {
+    inherit pname version;
```

Changing the `pname` will most likely make this not work here.
So we might need:
```suggestion
    pname = "PGPy";
    inherit version;
```